Regression/Checksum/benchmarks_json/ElectrostaticSphereEB.json

```diff
@@ -1,8 +1,8 @@
 {
   "lev=0": {
-    "Ex": 376867.5571515656,
-    "Ey": 376867.55715156574,
-    "Ez": 376867.5571515656,
+    "Ex": 277943.2629517972,
+    "Ey": 277943.2629517972,
+    "Ez": 277943.26295179717,
```

Comment:
Note that these values changed due to the use of `mlmg.getGradSolution` to solve for the E-field when embedded boundaries are present, which corrects the less accurate finite difference method used before (that didn't incorporate cut-cells).